### PR TITLE
feat: sync stream route labels to apisix

### LINKED ISSUE #775: request help: sync crd labels to apisix
### Issue description
i have some labels like group,project... in k8s crd,  i want to get the labels in apisix plugins. 
### Environment

* your apisix-ingress-controller version (output of `apisix-ingress-controller version --long`);
Version: 1.3.0
Git SHA: d1afc6e
Go Version: go1.17.2
Building OS/Arch: linux/amd64
Running OS/Arch: linux/amd64
* your Kubernetes cluster version (output of `kubectl version`);
* if you run apisix-ingress-controller in Bare-metal environment, also show your OS version (`uname -a`).

### PR DESCRIPTION
**Pull request by shreemaan-abhishek**
### Type of change:

New feature provided

### What this PR does / why we need it:

Closes #775

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ We have added partial support for Gateway API, which is not enabled by default,
 Ingress resources can now use all APISIX plugin configurations by setting the annotation `k8s.apisix.apache.org/plugin-config-name=xxx`.
 
 Please try out the release binaries and report any issues at
-https://github.com/apache/apisix-ingress-controller/issues.
+<https://github.com/apache/apisix-ingress-controller/issues>.
```
Comment (tao12345666333) [inline CHANGELOG.md:52]: This file is automatically generated and does not require modification.

File: test/e2e/suite-chore/admin_api.go [hunk 2]
```diff
@@ -18,10 +18,10 @@ package chore
 import (
 	"context"
 
+	v1 "github.com/apache/apisix-ingress-controller/pkg/types/apisix/v1"
```
Comment (tao12345666333) [inline test/e2e/suite-chore/admin_api.go:21]: No modifications are required here.

File: test/e2e/suite-features/external-sd.go [hunk 3]
```diff
@@ -23,10 +23,10 @@ import (
 	"os"
 	"time"
 
+	"github.com/apache/apisix-ingress-controller/pkg/config"
```
Comment (tao12345666333) [inline test/e2e/suite-features/external-sd.go:26]: ditto

**Review by tao12345666333 — COMMENTED** [review]
Thanks!

Could you please add e2e test case to cover this one?

**Comment by shreemaan-abhishek** [comment]
@tao12345666333 These changes were made when I ran `make update-all`

**Comment by tao12345666333** [comment]
Please add e2e test case to cover this one, Thanks!

**Comment by shreemaan-abhishek** [comment]
@tao12345666333 I did not find an example CRD yaml for `StreamRoute`. It is not even listed [here](https://github.com/apache/apisix-ingress-controller/tree/master/samples/deploy/crd/v1)

**Comment by shreemaan-abhishek** [comment]
Also, looks like there is some issue in the linter. There is absolutely no issue on my local 🤔

**Comment by shreemaan-abhishek** [comment]
Probably `StreamRoute` is just a `Route` CRD that has a `Stream` in it 🤔 

If that's true then ultimately, `StreamRoute` is just a `Route`. And [this PR](https://github.com/apache/apisix-ingress-controller/pull/1526) already addresses syncing `Route` CRD labels.

**Comment by tao12345666333** [comment]
Yes 

It means ApisixRoute 's `spec.stream`

**Comment by shreemaan-abhishek** [comment]
Then this PR should be closed. Please confirm.

**Comment by tao12345666333** [comment]
https://github.com/apache/apisix/blob/1a7fe40ba982c04e2a06901f942c5d4fc06084c8/apisix/schema_def.lua#L890-L917

According to the schema definition of APISIX, stream_route does not include labels, so I will close this. Thanks

**Pull request closed without merging**